packages/lfrqa/src/aviary/envs/lfrqa/env.py

```diff
@@ -118,68 +118,74 @@
 )
 
 
-class LFRQAPairwiseEvalEnv(GradablePaperQAEnvironment[dict]):
-    """Environment to evaluate paperqa's vs human's answers on Long Form RAG QA questions."""
+class LFRQAQuestion(MultipleChoiceQuestion):
+    gt_doc_ids: list[int]
 
-    def __init__(
-        self,
-        *args,
-        qid: str | UUID,
-        question: str,
-        human_answer: str,
-        gt_doc_ids: list[int],
-        pairwise_eval_llm: LLMModel | str = CommonLLMNames.GPT_4O.value,
-        **kwargs,
-    ):
-        kwargs["query"] = question
-        kwargs["docs"] = Docs()
-        super().__init__(*args, **kwargs)
+    @model_validator(mode="before")
+    @classmethod
+    def _validate_fields(cls, data: dict) -> dict:
+        if data.get("gold_doc_ids") and not data.get("gt_doc_ids"):
+            data["gt_doc_ids"] = data["gold_doc_ids"]
+        if isinstance(data["gt_doc_ids"], str):
+            data["gt_doc_ids"] = data["gt_doc_ids"].strip("[]").split(",")
+            data["gt_doc_ids"] = [int(_id) for _id in data["gt_doc_ids"]]
+        del data["gold_doc_ids"]
 
-        self.qid = qid
-        self.question = question
-        self.human_answer = human_answer
-        self.gt_doc_ids = gt_doc_ids
-        self.pairwise_eval_llm = pairwise_eval_llm
+        data["ideal_answer"] = data["answer"]
+        del data["answer"]
+
+        data["question_id"] = data["qid"]
+        del data["qid"]
+
+        data["options"] = []
+        data["prompt_without_options"] = True
 
-    def extract_best_answer_index(self, text: str) -> int:
+        return data
+
+    def _extract_best_answer_index(self, text: str) -> int:
         match = re.search(r"<rating>(\d+)</rating>", text)
         return int(match.group(1)) if match else 0
 
-    async def _evaluate_answer(self) -> dict:
-        """Pairwise evaluation of PaperQA vs Human answer."""
-        paper_search_ids = [int(doc.docname) for doc in self.state.docs.docs.values()]
-
-        pairwise_eval_llm = LiteLLMModel(name=self.pairwise_eval_llm)
-        pqa_answer = strip_citations(self.state.session.answer)
+    async def grade(  # type: ignore[override]
```

Comment:
Added this `type: ignore` bc now it returns a dict instead of a `MultipleChoiceEvaluation` as in the super